### ISSUE TITLE
[`blocks-dev`] Default value of Checkbox does not show up

### ISSUE THREAD

**Issue by abidlabs** [gradio-app/gradio#881]

**Comment by FarukOzderim** [comment]
Is this a checklist to mark when a component is testing for working without an issue?

**Comment by abidlabs** [comment]
That's honestly a good idea. Would probably subsume many of the individual issues that I created and easier to manage. Feel free to create it!

**Comment by abidlabs** [comment]
Not sure if this fits into this issue or a separate issue, but I noticed that the `default_value` for a `Checkbox` doesn't show up. (It does for `CheckboxGroup`, `Radio`, etc.)

**Comment by abidlabs** [comment]
@pngwn I think there are still some issues with Checkbox:

<img width="729" alt="image" src="https://user-images.githubusercontent.com/1778297/161813673-1ac69bc6-3d98-4ae7-9c53-2c16f0cc78fd.png">

* The `label` gets assigned to both the label and the associated text. The correct behavior is that the `default_value` parameter should be the associated text
* The Checkbox seems to pass in the default_value. I think the correct behavior is that it should pass in a boolean: True or False, depending on it's checked. 

Compare with CheckboxGroup, which is behaving correctly:

<img width="759" alt="image" src="https://user-images.githubusercontent.com/1778297/161813912-3daee717-46b5-4201-9ada-09b883891175.png">

**Comment by pngwn** [comment]
> The label gets assigned to both the label and the associated text. The correct behavior is that the default_value parameter should be the associated text

> 
The Checkbox seems to pass in the default_value. I think the correct behavior is that it should pass in a boolean: True or False, depending on it's checked.

I don't think this is correct unless i'm misreading the API. `Checkbox` only has a mechanism to pass in one 'label'. So the duplication is just because no associated text is passed in at all [from the docs](https://gradio.app/docs/#i_checkbox):

```py
gradio.inputs.Checkbox(self, default=False, label=None, optional=False)
```

So passing the `default_value` is correct unless things have change significantly from the original API (I thought checkbox was mostly the same).

I've modified the UI a little so the Checkbox appears on its own with no additional text, this is consistent with current usage. We canm discuss if an additional piece of associated text makes sense.

**Comment by abidlabs** [comment]
One very small part of #897 is left:

> The output textbox size does not dynamically change size according the output as it used to (see screenshot)

Here's a screenshot:

<img width="793" alt="image" src="https://user-images.githubusercontent.com/1778297/162053789-ce03a5a0-86d5-47eb-8202-ab842274be08.png">

Everything else looks great!!

**Comment by pngwn** [comment]
Closed by #929.
